microsoft-edge/dev-videos/index.md

```diff
@@ -30,265 +30,837 @@ To add a new video:
 
+Many people who are blind or low vision experience the web primarily through a screen reader: an assistive technology that reads the content of each page aloud.  Screen readers depend on having image labels (alternative text or "alt text") provided that allows them to describe visual content - like images and charts - so the user can understand the full content of the page.  Alt text is critical for making the web accessible, yet it’s often overlooked.  More than half of the images processed by screen readers are missing alt text. 
 
-<!-- ====================================================================== -->
-## DevTools - What's New in DevTools 99
-
-_Published on 21 Mar 2022._
+See also:
+* [Appears to say: Microsoft Edge now provides auto-generated image labels](https://blogs.windows.com/msedgedev/2022/03/17/appears-to-say-microsoft-edge-auto-generated-image-labels/) - blog post
 
-[![Thumbnail image for the DevTools What's New in 99 video](./index-images/devtools-whatsnew-99.png)](https://www.youtube.com/watch?v=Z5-tEE_cNTo)
 
-Learn more about our latest announcements from the Microsoft Edge DevTools team at [What's New in DevTools 99](../devtools-guide-chromium/whats-new/2022/03/devtools.md).
+<!-- ====================================================================== -->
+## Web Platform - The EyeDropper API
 
+November 22, 2021
 
-<!-- ====================================================================== -->
-## DevTools - What's New in DevTools 98
+[![Thumbnail image for the Eye Dropper API video](./index-images/eyedropper-api.png)](https://www.youtube.com/watch?v=XZUEnUbI7dE)
 
-_Published on 23 Feb 2022._
+The Microsoft Edge team specified and implemented the new EyeDropper API in collaboration with the Chromium open-source project. Provide feedback at [Issues - WICG/eyedropper | github.com](https://github.com/WICG/eyedropper-api/issues).
 
-[![Thumbnail image for the DevTools What's New in 98 video](./index-images/devtools-whatsnew-98.png)](https://www.youtube.com/watch?v=HpaRDwU_AZI)
+Many creative applications enable users to pick colors from parts of an app window or even from the entire screen, typically using an eyedropper metaphor. The EyeDropper API enables authors to use a browser-supplied eyedropper in the construction of custom color pickers on the web.
 
-Learn more about our latest announcements from the Microsoft Edge DevTools team at [What's New in DevTools 98](../devtools-guide-chromium/whats-new/2022/03/devtools.md).
+See also:
+* [Picking colors of any pixel on the screen with the EyeDropper API | web.dev](https://web.dev/eyedropper/)
+* [EyeDropper API - Web APIs | MDN developer.mozilla.org](https://developer.mozilla.org/docs/Web/API/EyeDropper_API)
 
 
 <!-- ====================================================================== -->
-## DevTools - What's New in DevTools 97
+## Microsoft Search - How developers can add value to search within the workplace (Microsoft Build 2021)
```

Comment:
This one is not related to Edge, we should remove it.

Reply:
removed